web/web/api/v1/meter.py

```diff
@@ -102,25 +106,40 @@ def add_meter_info():
     '''Add new meter to database'''
 
     #retrieve input
-    meter_id = request.form.get('meter_id')
-    utility_id = int(request.form.get('utility_id'))
-    service_location_id = request.form.get('service_location_id')
-    feeder = request.form.get('feeder')
-    substation = request.form.get('substation')
-    meter_type = request.form.get('meter_type')
-    is_active = request.form.get("is_active").upper() == "TRUE"
-    is_archived = request.form.get("is_archived").upper() == "TRUE"
+    new_meter = request.get_json()
+
+    meter_id = new_meter['meter_id']
+    utility_id = new_meter['utility_id']
+    service_location_id = new_meter['service_location_id']
+    feeder = new_meter['feeder']
+    substation = new_meter['substation']
+    meter_type = new_meter['meter_type']
+    is_active = new_meter['is_active'].upper() == "TRUE"
+    is_archived = new_meter['is_archived'].upper() == "TRUE"
 
     try:
-        service_location = ServiceLocation.query.filter_by(service_location_id=service_location_id).one()
-        meter = Meter(meter_id=meter_id, utility_id=utility_id, service_location_id=service_location.service_location_id, feeder=feeder, substation=substation, meter_type=meter_type, is_active=is_active, is_archived=is_archived)
+        #TO DO: insert error handling to make sure meter id doesn't already exist
+        
+        #checks if service location exists
+        service_location=ServiceLocation.query.filter_by(service_location_id=service_location_id).one()
+        
+        meter = Meter(meter_id=meter_id, 
+                      utility_id=utility_id, 
+                      service_location_id=service_location.service_location_id, 
+                      feeder=feeder, 
+                      substation=substation, 
+                      meter_type=meter_type, 
+                      is_active=is_active, 
+                      is_archived=is_archived)
+        
         db.session.add(meter)
         db.session.commit()
+        
+        return jsonify({'success': meter.meter_id})
 
     except (MultipleResultsFound,NoResultFound) as e:
-        #no results or multiple results found 
+        #no results or multiple results found for service location
         print(e)
-        return 'error'
```

Comment:
Let's return an actual json error response.

We'll eventually write a response wrapper object, but for now, something like:
```python
return jsonify({'error': 'my error was....'})
```
should work